app/models/building.rb

```diff
@@ -0,0 +1,8 @@
+class Building < ActiveRecord::Base
+  has_many :tenants, class_name: User.name
```

Comment:
I think that if multiple tenants are allowed in an apartment at the same time (as opposed to only one current tenant per apartment), we will want to group the aggregate stats by apartment and present everything by apartment, so in most cases the join between Apartment <-> Tenant won't need to be done.